### PR TITLE
Use string interpolation

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

File: Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs [hunk 1]
```diff
@@ -45,7 +45,8 @@ public class RemoteConsoleCommandLine : CommandLine {
                 // All bets are off while accessing the remote runtime. So we catch all exceptions.
                 // However, in most cases, we only expect to see RemotingException here.
                 if (!(exceptionDuringHandling is RemotingException)) {
-                    Console.WriteLine(String.Format("({0} thrown while trying to display unhandled exception)", exceptionDuringHandling.GetType()), Style.Error);
```
Comment (slide) [inline Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs]: This one could have just removed the String.Format and used the WriteLine overload which takes params.

File: Src/Microsoft.Scripting/Runtime/DynamicStackFrame.cs [hunk 2]
```diff
@@ -54,14 +54,10 @@ public class DynamicStackFrame {
...
-        public override string ToString() {
-            return string.Format(
-                "{0} in {1}:{2}, {3}",
-                _funcName ?? "<function unknown>",
-                _filename ?? "<filename unknown>",
-                _lineNo,
-                (_method != null ? _method.ToString() : "<method unknown>")
-            );
+        public override string ToString()
+        {
```
Comment (slide) [inline Src/Microsoft.Scripting/Runtime/DynamicStackFrame.cs]: Nitpick- brace should be on same line as method name

File: Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs (hunk 1 above)
Comment (gpetrou) [inline Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs]: Style.Error is still there. Are you talking about something else?

Comment (slozier) [inline Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs]: He probably though this was System.Console.WriteLine.

Comment (slide) [inline Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs]: Ah, you are correct. Apparently I need to look at filenames :-)

**Pull request merged**